### DISCUSSION TITLE
BookType Enum-Question

### DISCUSSION THREAD

**Discussion by sboyd** [RehanSaeed/Schema.NET#25] [Q&A]
First off thanks Mr. @RehanSaeed for creating this library!

I've got a question about this BookTypes.
We're using this library to create structured data and when I set the 
BookFormat to BookFormatType.EBook,

When it gets returned it has the value of 1. I'm going to have to explain why EBook has the value of 1 and why that's ok. Is there any documenation that you can point me to that shows 1 is a value value for EBook?

`[EnumMember(Value = "http://schema.org/EBook")]
EBook`

Thanks! --Steve

**Comment by RehanSaeed** [comment]
All documentation can be found on schema.org. In your case you want https://schema.org/BookFormatType

Feel free to reopen if you have more questions.

**Comment by sboyd** [comment]
Thank you for pointing me to the documentation.

I'd spent quite a bit of time there, before reaching out so as not to waist your time, trying to see if I could figure out how you knew that 

EBook gets a value of 1, but I don't see any sort of mapping.

**Comment by RehanSaeed** [comment]
Well it's an enum and enum's are just integers.
